sunpy/map/map_factory.py

```diff
@@ -60,6 +61,10 @@ class MapFactory(BasicRegistrationFactory):
     composite : `bool`, optional
         Return a `sunpy.map.CompositeMap` object comprised of all the parsed maps.
 
+    filetype : `str`, optional
+        Manually specify the filetype.
+        Supported extensions are ('fts', 'fits', 'jp2', 'j2k', 'jpc', 'jpt', 'fz', 'f0')
```

Comment:
I would rather avoid adding this keyword if we can avoid it.

Reply:
That's doable,

But, suppose a user has a file with an unknown type,
- they can either read the header and supply the filetype directly, bypassing the whole detection stack
- or they run the detection once and supply the filetype in the future,

assuming the user doesn't wanna rename the file;
So wouldn't the filetype keyword be valuable in this context?

Reply:
I feel like a user should know what type their file is.

Reply:
That's fair